conf/glancesync.conf

```diff
@@ -83,15 +83,25 @@ metadata_condition = image.is_public and\
 # user variables are synchronised.
 metadata_set = nid , type, sdc_aware, nid_version
 
-# if true, ignore public images of other tenants. That is, an image is upload
-# even when a image with the same name and content exist in the regional
-# server, if this image is not owned by the tenant specified in the credential.
-# Usually, it is convenience to set this value to True. It some images were
-# owned by other tenants, a better option is to change their owner.
+# When the software asks for the list of images in a region, it gets both the
+# images owned by the tenant and the public images owned by other tenants.
+# If this parameter is true (the default and recommended value), only the
```

Comment:
Yes, the README.rst uses glancesync.conf file as example, therefore the changes are
in the two files (or glancesync.conf is a copy ready to use of the configuration sample of README.rst)